### PR TITLE
Bump XCode to latest

### LINKED ISSUE #53: Error when I run a project
no such module 'awscognitoidentityprovider'
Hi, I want to try the Kinesis Video Stream sample for ios. when I try to build the app, I got this error. I have been removed Pods dir and rerun pod install multiple times or recreate project still get the same error. I really appreciate any help. Thank you
caf65e94fdfd286597617b2b8846021
This is my pod file list
582862da195147b011500b08fa73198

### LINKED ISSUE #62: About supporting higher versions of xcode
Please tell me when to update and upgrade. The currently supported version of xcode is too low. We need to support a higher version of xcode. At present, we are using the version of xcode 14.2. At present, there is no way to test the application function and subsequent development. All work is suspended. The stuck point is here. Please provide a feasible solution and update plan

### PR DESCRIPTION
**Pull request by sirknightj**
*Issue #, if available:*

*Description of changes:*
* Repair CI
* Add more XCode versions, Mac versions, ios versions, to the CI
* Upgrade XCode version to latest
* Update README instructions (with some troubleshooting steps as well)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Swift/KVSiOSApp/AppDelegate.swift [hunk 1]
```diff
@@ -97,7 +97,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         static func lockOrientation(_ orientation: UIInterfaceOrientationMask, andRotateTo rotateOrientation: UIInterfaceOrientation) {
             self.lockOrientation(orientation)
-            UIDevice.current.setValue(rotateOrientation.rawValue, forKey: "orientation")
+//            if #available(iOS 16, *) {
```
Comment (disa6302) [inline Swift/KVSiOSApp/AppDelegate.swift:100]: Meant to comment?

Comment (sirknightj) [reply Swift/KVSiOSApp/AppDelegate.swift:100]: Yes

**Review by disa6302 — APPROVED** [review]
Can we verify if we really need to commit these 2 files?

`Swift/AWSKinesisVideoWebRTCDemoApp.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist`
`Swift/AWSKinesisVideoWebRTCDemoApp.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings`

If not, lets remove it. Otherwise LGTM.

**Comment by sirknightj** [comment]
Looks like we want to merge stuff in `/xcshareddata`, just don't want to include stuff in `/xcuserdata` (which is already removed)

**Pull request merged**